components/model/model-card.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { IModelData } from '@/types/model';
+import { useTranslation } from 'react-i18next';
+import SentimentSatisfiedAltIcon from '@mui/icons-material/SentimentSatisfiedAlt';
+import SentimentVeryDissatisfiedIcon from '@mui/icons-material/SentimentVeryDissatisfied';
+import StopCircleIcon from '@mui/icons-material/StopCircle';
```

Comment:
mui will be deprecated soon，please choose the icon in `@antd/icons`

Reply:
mui icon is more abundant then antd, how about continuing using it. component can change to antd.